civilcode.shrc

```diff
@@ -4,9 +4,19 @@ alias dea="docker-compose exec application"
 alias deat="docker-compose exec -e MIX_ENV=test application"
 alias deam="docker-compose exec application mix"
 alias deatm="docker-compose exec -e MIX_ENV=test application mix"
+alias dcs="docker-compose stop"
+alias dcu="docker-compose up -d"
 
 alias elixir-format="watchman -- trigger ./ format '**/*.ex' '**/*.exs' '**/*.default' -- mix format"
 
+function deamtw() {
+	docker-compose exec application mix test.watch
+}
+function deamtwl() {
+	app="$1"; shift
+	docker-compose exec application mix cmd --app $app mix test.watch $*
+}
```

Comment:
Trying to work out what the `l` stands for :-)

Reply:
`local`.. maybe `deamtwa` for `child app`?